### PR TITLE
LK training into dev branch request

### PR DESCRIPTION
**Pull request by LonjezoK**
## Proposed changes

Big picture changes? Not sure really a small picture because I only changed one file but hey butterfly effect huh I guess one small change can change everything.

Text changes mainly, that's it.

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ X] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ X] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ X] All of your unit tests pass
- [ X] Your code is clear and easy to understand
- [X ] You have documented any new features
- [X ] They can run the code you have written
- [ X] Your changes do not break any other functionality

Going to get SK to review this pull request. 
@TD-Semanur-karakurt 

#201 is the issue I raised I think.

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -1,5 +1,10 @@
 This is a random text file. You can add and remove things to it to test how git works.
 
 
-This is some text.
+T
```
Comment (DIPAD-Fran-Bryden) [inline text_file.txt:4]: Think you have deleted this by accident!

File: text_file.txt [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
 This is some other text.
+
+Lonjezo added this text
+He also removed the text on line 4.
+
```
Comment (DIPAD-Fran-Bryden) [inline text_file.txt:9]: Unnecessary line break!

**Review by DIPAD-Fran-Bryden — APPROVED** [review]

File: text_file.txt [hunk 3]
```diff
@@ -1,5 +1,10 @@
 This is a random text file. You can add and remove things to it to test how git works.
 
 
-This is some text.
```
Comment (TD-Semanur-karakurt) [inline text_file.txt:4]: I think you deleted this by accident. (haha joking)

File: text_file.txt [hunk 4]
```diff
@@ -1,5 +1,10 @@
...
+T
 This is some other text.
+
+Lonjezo added this text
+He also removed the text on line 4.
```
Comment (TD-Semanur-karakurt) [inline text_file.txt:8]: I agree.

**Review by TD-Semanur-karakurt — COMMENTED** [review]
Great code!

**Pull request closed without merging**